Collapsible.js

```diff
@@ -186,9 +186,12 @@ export default class Collapsible extends Component {
       animating,
     } = this.state;
     const hasKnownHeight = !measuring && (measured || collapsed);
-    const style = hasKnownHeight && {
+    const style = hasKnownHeight ? {
```

Comment:
@truph01 Simply setting the styles by omitting the `hasKnownHeight` won't work?

```js
const style = {
  overflow: 'hidden',
  height: height,
}
```

Reply:
Fixed in 13af1dd